src/index.js

```diff
@@ -42,10 +42,11 @@ const MainAppWithApolloProvider = () => {
   const [client, setClient] = useState();
   const [initialGlobalSettings, setInitialGlobalSettings] = useState({
     filter: {},
+    hdvt: {},
+    navigation: 'tab',
     sections: {},
     settings: {},
-    widgets: [],
-    hdvt: {}
+    widgets: []
   });
   const [initialListTypesSettings, setInitialListTypesSettings] = useState({});
   const [initialLocationSettings, setInitialLocationSettings] = useState({});
```

Comment:
why is it needed to write all this again if its already present in the SettingsProvider.js file

Reply:
it's a state variable here not related to the settings provider. at the end this will be passed to the settings provider. there should be a way to remove one or the other, but it doesn't hurt so far.